resources/assets/lib/chat/conversation-list.tsx

```diff
@@ -31,6 +32,7 @@ export default class ConversationList extends React.Component {
     return channels.map((conversation) => <ConversationListItem key={conversation.channelId} channelId={conversation.channelId} />);
   }
 
+  @action
```

Comment:
This `action` doesn't seem to do anything?; also looks like `nonPmChannels` and `pmChannels` in `ChannelStore` should be `@computed.struct` instead of just `@computed`.